### PR TITLE
Add happy cases for functions tested in src/utils/validation.test.ts.

### LINKED ISSUE #464: Add happy cases to tests of validation functions
# ISSUE

## Overview

We are missing the happy cases (success cases) for functions tested in `src/utils/validation.test.ts`.

### PR DESCRIPTION
**Pull request by AmberSwieczkowski**
Closes #464.

### CODE REVIEW

File: src/utils/validation.test.ts [hunk 1]
```diff
@@ -91,6 +139,12 @@ describe("validateUint8Array", () => {
       "Expected parameter 'test' to be type 'Uint8Array', was type 'string', value '123'"
     );
   });
+
+  const cases = [new Uint8Array([17, -45.3])];
```
[the remark this answers is not in the document]
Comment (amberm31) [reply src/utils/validation.test.ts]: bitmoji.

**Review by m-cat — DISMISSED** [review]
Very impressive!

**Review by m-cat — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]
Nice!

**Comment by kwypchlo** [comment]
@AmberSwieczkowski you need your commit to be signed though, otherwise github will not let us merge the pull request. 

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by AmberSwieczkowski** [comment]
I had to open a new pull request to add the signature verification. It's #482.

**Pull request closed without merging**